src/components/EditUser.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const EditUser = props => {
+  if (props.selectedUser) {
+    return (
+      <div>
+        <div className="editfields">
+          <div>
+            <label>id: </label>
+            {props.addingUser
+              ? <input
+                  type="number"
+                  name="id"
+                  placeholder="id"
```

Comment:
Eventually we probably don't want our users to add their own 'id' :) But this is fine for now!